.travis.yml

```diff
@@ -16,21 +21,39 @@ before_install:
   - 'npm config set registry http://registry.npmjs.org/'
   - 'export DISPLAY=:99.0'
   - 'sh -e /etc/init.d/xvfb start'
+  - 'sudo add-apt-repository -y ppa:webupd8team/java'
+  - 'sudo apt-get update'
+  - 'sudo apt-get install -y libappindicator1 fonts-liberation'
+  - 'sudo apt-get install -y oracle-java8-installer'
+  - 'sudo apt install -y oracle-java8-set-default'
+  - 'wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb'
+  - 'sudo dpkg -i google-chrome*.deb'
+  - 'npm install -g cypress-cli' #test using cypress cli for integrationg testing
+  - 'wget https://seqwaremaven.oicr.on.ca/artifactory/collab-release/io/dockstore/dockstore-webservice/1.1-rc.2/dockstore-webservice-1.1-rc.2.jar'
```

Comment:
Change this to 1.1 and make it an environment variable so it is easy to change https://docs.travis-ci.com/user/environment-variables/